Core/TimberApi/BottomBarSystem/BottomBarSystemConfigurator.cs

```diff
@@ -0,0 +1,21 @@
+using Bindito.Core;
+using TimberApi.BottomBarSystem.Patchers;
+using TimberApi.BottomBarUISystem;
+using TimberApi.ConfiguratorSystem;
+using TimberApi.SceneSystem;
+
+namespace TimberApi.BottomBarSystem
+{
+    [Configurator(SceneEntrypoint.InGame)]
```

Comment:
Maybe add `SceneEntryPoint.MapEditor` so that buttons can be added to the mapeditor?

Reply:
There are few things that holding me back.
- Probably also needing to maintain all kinds of different factories for MapEditor tools
- It might not work (at least the bottombat implementation)